assets/json-schema-faker.js

```diff
@@ -24430,6 +24430,10 @@ function extend() {
       return dateTimeGenerator().slice(11);
   }
 
+  const FRAGMENT = '[a-zA-Z][a-zA-Z0-9+-.]*';
+  const URI_PATTERN = `https?://{hostname}(?:${FRAGMENT})+`;
+  const PARAM_PATTERN = '(?:\\?([a-z]{1,7}(=\\w{1,5})?&){0,3})?';
```

Comment:
also comment where you've referenced these from - unless there's a standard source to back these up, these will need exhaustive testing

Reply:
approved the PR, but add comments here

Reply:
@abhijitkane Yes, the [source](https://github.com/json-schema-faker/json-schema-faker/blob/master/src/lib/generators/coreFormat.js) for this is official json-schema-faker latest version. I will add comment for the same.